### ISSUE TITLE
Fuses not set on blank IC

### ISSUE THREAD

**Issue by vsavic** [adafruit/Standalone-Arduino-AVR-ISP-programmer#6]
Fuses are not being set on blank ATMega328P-PU. Program report error during the fuse verification process, followed by constant beeping sound. Once fuses are set (with some other tool (not really standalone if we need additional tool for that)), program works.

**Comment by ladyada** [comment]
try latest commit!

**Comment by vsavic** [comment]
Now it works in 98% of the time. I tried about 10 blank ICs, and just a few times when I pressed the button for the first time, I got the same beeping error and same error as mentioned above, but when I power it off and on again it was working again. So, your latest commit looks like it fixed it, but not 100% looks like there's some minor issue now somewhere else. It might be a good idea to reset that error state after X seconds so we can try one more time by pressing the button instead of powering off and on again.
